src/intrinsics.ts

```diff
@@ -150,9 +152,11 @@ export function linkIntrinsics(
         const blue = blueIntrinsics[name];
         const red = redIntrinsics[name];
         // new intrinsics might not be available in some browsers, e.g.: AggregateError
-        if (!isUndefined(blue)) {
+        if (isObjectLike(blue)) {
```

Comment:
can you provide more details about why this change? the change makes sense... but I wonder which one was the offender here that makes you change the code from defined to is object like?

Reply:
In the case we really want to know if the value can be a valid weak map key which means more than just `!isUndefined()`. So it's just a more accurate check... the added check for `isObjectLike(blue.prototype)` is for the case of `Proxy` which has no prototype.